src/components/grid/Grid.styles.tsx

```diff
@@ -96,18 +96,38 @@ Container.defaultProps = {
   mx: 'auto',
 }
 
-export const FlexColumn = styled(Column)`
-  display: block;
-  ${mediaQueries.md} {
-    display: flex;
+const keyFrame = keyframes`
+  0% {
+    transform: scale(0);
+    opacity: 0;
+  }
+  100% {
+    transform: '';
+    opacity: '';
   }
 `
-
-export const StyledFlipMove = styled(FlipMove)`
-  display: flex;
-  flex-wrap: wrap;
-  flex-basis: 100%;
+// styled componets V4 requeries css helper func for keyframes
+const keyFrameHelper = css`
+  animation: ${keyFrame} 360ms ease;
 `
+type FlexColumnProps = {
+  animation: boolean
+  children: object
+}
+export const FlexColumn = ({
+  animation,
+  children,
+  ...props
+}: FlexColumnProps) => {
+  const FlexInternal = styled(Column)`
```

Comment:
Should the FlexInternal const be declared inside the component?

Reply:
Ya your right it makes more sense to make a specific one for the events listing rather than add the functionality to the Flexcolumn component as it is used quite a bit in the project. I'll fix that.

I need the internal const because of ESLint conflicting with styled-components V4 and it's requirement for a css helper func for keyframes. It's the only workaround that I found at the moment